CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [v1.2.3](https://github.com/ably/ably-ruby/tree/v1.2.3)
+
+[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.2.2...v1.2.3)
+
+**Closed issues:**
+
+- Incorrect ProtocolMessage\#connection\_details object \(overwrites original connection\_details send on CONNECTED state\) [\#377](https://github.com/ably/ably-ruby/issues/377)
+
+**Merged pull requests:**
```

Comment:
I would thin out this section to remove issues that don't directly improve the customer experience of using this SDK... e.g. remove test specs and test observability.

Reply:
Yeah good point, I've removed the fluff and structured it a bit better now